core/src/avm2/globals/flash/net/URLStream.as

```diff
@@ -0,0 +1,118 @@
+package flash.net {
+    import flash.events.Event;
+    import flash.events.EventDispatcher;
+    import flash.utils.Endian;
+    import flash.utils.IDataInput;
+    import flash.utils.ByteArray;
+    import flash.events.Event;
+    import flash.events.IOErrorEvent;
+    import flash.events.SecurityErrorEvent;
+    import flash.events.ProgressEvent;
+    import flash.events;
+    import __ruffle__.stub_method;
+    import __ruffle__.stub_getter;
+    import __ruffle__.stub_setter;
+
+    public class URLStream extends EventDispatcher implements IDataInput {
+        private var _endian:String = Endian.BIG_ENDIAN;
+        private var _loader:URLLoader = new URLLoader();
+
+        public function URLStream() {
+            this._loader.dataFormat = URLLoaderDataFormat.BINARY;
```

Comment:
Can you drop a comment in the constructor to explain that this is temporary implementation and not what we expect this to do in the future? Even just a `// HACK, see https://github.com/ruffle-rs/ruffle/pull/11046` would be sufficient

Reply:
I added a comment to the `URLLoader` field.